tests/test-mongoc-command-monitoring.c

```diff
@@ -1745,6 +1745,144 @@ test_killcursors_deprecated (void)
 }
 
 
+static void
+command_failed_reply_command_started_cb (const mongoc_apm_command_started_t *event)
```

Comment:
does it buy us anything to register a command-started callback? i think we're only testing the new feature below, in the command-failed callback.

Reply:
The only reason I used a command-started callback was to assert that the command_name was to assert that `test->cmd_name == mongoc_apm_command_failed_get_command_name (event)`. I removed that to remove the command-started callback.